test/sql/include/plan_expand_hypertable_query.sql

```diff
@@ -141,19 +141,51 @@ SELECT * FROM cte ORDER BY value;
 
 \qecho test constraint exclusion for constraints in ON clause of JOINs
 
+-- Turn on chunk exclusion so that we can compare with how
+-- PostgreSQL does exclusion
+SET constraint_exclusion=on;
+SHOW constraint_exclusion;
+
 \qecho should exclude chunks on m1 and propagate qual to m2 because of INNER JOIN
+SET timescaledb.enable_constraint_exclusion = off;
```

Comment:
I know these are existing test cases. But it would help to have different tables for JOIN , esp. RIGHT JOIN and LEFT JOIN cases. That helps easily identify when the chunks are being excluded and when they are not.

Reply:
I thought about it, but I don't think it makes a practical difference. From the planners point of view, these are different rels (RelOptInfos) that just happens to point to the same underlying table. So, even if the table is the same, it can exclude differently depending on the planned rel, as shown in the examples (i.e., exclusion can happen on m1 but node m2, even if same table).

Reply:
Yes, of course, it works the same way. It is easy to identify the tables in the explain plan output when they have different chunk names.

Reply:
Ok, let me see what I can do

Reply:
Ok, this is now implemented.